docs/general/ad-filtering/create-own-filters.md

```diff
@@ -1006,6 +1007,20 @@ If you want the rule not to be applied to certain apps, start the app name with
 >
 > Only AdGuard for Windows, Mac, Android are technically capable of using rules with `$app` modifier.
 
+#### **`$method`** {#method-modifier}
+
+This modifier limits the scope of the rule to requests that use the specified set of HTTP methods. Negated methods are allowed. The set of allowed methods is the same as in Chrome's [declarativeNetRequest](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/#type-RequestMethod).
```

Comment:
will it work only in chrome? and only if chrome v91 or newer?

Reply:
No, only the what you can type after `method=` is defined by this list :) We are still thinking about lifting this restriction, please ignore for now